README.md

```diff
@@ -43,27 +43,29 @@ import (
     "github.com/Eppo-exp/golang-sdk/v4/eppoclient"
 )
 
-var eppoClient = &eppoclient.EppoClient{}
+var eppoClient *eppoclient.EppoClient
```

Comment:
This code example was not valid - the `EppoClient` cannot be instantiated in the way it was documented. Instead we create a global variable and declare its type; it is currently `nil`.